lib/ZumoOta/src/BootloaderCom.cpp

```diff
@@ -75,8 +79,9 @@ class IntProgCmdProvider: public CmdProvider {
      * @param rsp Reference to a pointer to the next response.
      * @return True if there is a next command and next response, false otherwise.
      */
-    virtual bool next( const CommandInfo *& cmd,  const ResponseInfo *& rsp)
+    virtual bool next(  CommandInfo *& cmd,   ResponseInfo *& rsp)

```

Comment:
the const was there before by purpose. See comment at line 344 below. the process() routine shall not modify commands, the provider need to feed with with ready to use ones.